packages/babel-helpers/src/helpers.js

```diff
@@ -87,12 +87,16 @@ helpers.asyncIterator = defineHelper(`
   }
 `);
 
-helpers.asyncGenerator = defineHelper(`
-  function AwaitValue(value) {
+helpers.AwaitValue = defineHelper(`
+  export default function _AwaitValue(value) {
     this.value = value;
   }
+`);
+
+helpers.AsyncGenerator = defineHelper(`
+  import AwaitValue from "AwaitValue";
 
-  function AsyncGenerator(gen) {
+  export default function AsyncGenerator(gen) {
```

Comment:
Is this helper only used by `wrapAsyncGenerator`? If so, we can move it there.

Reply:
I believe it is. I've splitted them, because it made sense to me to keep AsyncGenerator separate as its standalone class. Can move it back in ofc, no problem